components/Calendar.js

```diff
@@ -68,6 +75,12 @@ export default class Calendar extends Component {
     weekStart: 1,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.onWeekRowLayout = this.onWeekRowLayout.bind(this);
+  }
+
```

Comment:
Fixed in https://github.com/christopherdro/react-native-calendar/pull/73/commits/81260fcbdc9bf8d2243955156dadbcacbc063973